client/app/styles/main.styl

```diff
@@ -732,20 +732,20 @@ body
         box-sizing: border-box
         width: 180px
         height: 180px
-        padding: 20px 20px 10px 20px
+        padding: 25px 20px 10px 20px
         margin-right: 10px
 
         img.svg
-            border-radius: 200px
-            padding: 10px
-            width:140px
-            height:140px
+            border-radius 200px
+            padding 15px
+            width 130px
+            height 130px
 
```

Comment:
Even if I prefer the stylus syntax w/o colons, I think it's better to stick to previous conventions and keep them in our current files. It's just for consistency, tell me if you think I'm too much rigid :smile:.